backend/main-service/src/main/java/kkakka/mainservice/coupon/domain/MemberCoupon.java

```diff
@@ -0,0 +1,19 @@
+package kkakka.mainservice.coupon.domain;
+
+import javax.persistence.EmbeddedId;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "MemberCoupon")
+public class MemberCoupon {
+
+    @EmbeddedId
+    private MemberCouponId memberCouponId;
```

Comment:
```suggestion
    private MemberCouponId memberCouponId;
    
    @MapsId(value = "memberId")
    @ManyToOne(fetch = FetchType.LAZY)
    private Member member;

    @MapsId(value = "couponId")
    @ManyToOne(fetch = FetchType.LAZY)
    private Coupon coupon;
```

이런 식으로 `MemberCoupon` 과 `Member`, `Coupon` 사이에 연관관계 매핑을 통해 `MemberCoupon` 조회할 때 fetch join 으로 정보를 한 번에 가져오면 좋을 것 같아요~!